easybuild/easyconfigs/s/SSW/SSW-1.2.4-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,48 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'SSW'
+version = '1.2.4'
+
+homepage = 'https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library'
+description = """SSW is a fast implementation of the Smith-Waterman algorithm, which uses the
+ Single-Instruction Multiple-Data (SIMD) instructions to parallelize the algorithm at the
+ instruction level. SSW library provides an API that can be flexibly used by programs written in
+ C, C++ and other languages. We also provide a software that can do protein and genome alignment
+ directly. Current version of our implementation is ~50 times faster than an ordinary
+ Smith-Waterman. It can return the Smith-Waterman score, alignment location and traceback path
+ (cigar) of the optimal alignment accurately; and return the sub-optimal alignment score and
+ location heuristically.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/mengyao/Complete-Striped-Smith-Waterman-Library/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-1.2.4_build-cpp-lib.patch']
+checksums = [
+    'a87b1904368bffe051ab6ea538543ec1520473a5d6d94204bd6fa8e39d0cf336',  # v1.2.4.tar.gz
+    '1513b57036d671e84921c46f24699f9d4bfc8e587811498d72c19c64942320c5',  # SSW-1.2.4_build-cpp-lib.patch
+]
+
+builddependencies = [('binutils', '2.36.1')]
+
+dependencies = [('zlib', '1.2.11')]
+
+start_dir = 'src'
+
+buildopts = 'default'
+
+files_to_copy = [
+    (['ssw_test'], 'bin'),
+    (['*.%s*' % SHLIB_EXT], 'lib'),
+    (['*.h'], 'include/ssw'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ssw_test', 'lib/libssw.%s' % SHLIB_EXT],
+    'dirs': [],
+}
```

Comment:
@SIB-software would you consider adding `sanity_check_commands`?

Reply:
There is no `sanity_check_commands` in the previous SSW-1.1.1 EB files.

I don't really know this tool, I just need it as requirement.
The simplest `sanity_check_commands` I see could be to display the help message but it does not exit with a 0 status code. So `ssw_test || true` would not be a very useful check.

@verdurin What do you think?

Reply:
Or something like `ssw_test 2>&1 | grep 'Usage:'`  ?

Reply:
Yes, that's a commonly-used approach.
I'm just trying to add these as easyconfigs are updated, where possible, as they're a nice extra check.
You are free to say it's too much work.